### PR TITLE
Remove YDoc subclass and use YDoc.observe_after_transaction

### PR DESCRIPTION
**Pull request by davidbrochart**

### CODE REVIEW

File: ypy_websocket/yutils.py [hunk 1]
```diff
@@ -63,3 +66,33 @@ def get_messages(message):
 
 def get_message(message):
     return next(get_messages(message))
+
+
+def put_updates(update_queue: asyncio.Queue, ydoc: Y.YDoc, event) -> None:
+    update = Y.encode_state_as_update(ydoc, event.before_state)
```
Comment (davidbrochart) [inline ypy_websocket/yutils.py:72]: @Waidhoferj this gives a `RuntimeError: Already borrowed`.
I'm not sure I'm doing things right, isn't there another way to get the updates corresponding to the transaction?

Comment (davidbrochart) [reply ypy_websocket/yutils.py:71]: @Waidhoferj @dmonad any feedback?

Comment (Waidhoferj) [reply ypy_websocket/yutils.py:71]: Sorry I haven't gotten back to this. I'll take a look this weekend.

Comment (Waidhoferj) [reply ypy_websocket/yutils.py:71]: Isolated [the issue](https://github.com/y-crdt/ypy/issues/69). Will work on a fix

Comment (davidbrochart) [reply ypy_websocket/yutils.py]: Thanks for the fix @Waidhoferj, I checked that it works locally.
Could we get a ypy release?

**Pull request merged**